infoschema/infoschema.go

```diff
@@ -102,9 +102,8 @@ type InfoSchema interface {
 }
 
 // Information Schema Name.
-const (
-	Name      = util.InformationSchemaName
-	LowerName = util.InformationSchemaLowerName
+var (
+	Name = util.InformationSchemaName.O
```

Comment:
Can we remove this global variable and replace it with `util.InformationSchemaName.O`?